UPGRADE-2.1.md

```diff
@@ -24,3 +24,86 @@
 
 1. The `price`, `original_price`, `minimum_price` hookables from `'sylius_admin.product.update.content.form.sections.channel_pricing'`
    hook have been deprecated and disabled. Now these templates are located in `'sylius_admin.product.create.content.form.sections.channel_pricing.info'`.
+
+### Assets
+
+#### Overview of Changes
+Sylius has modernized its asset management system with these key improvements:
+- Bundle-prefixed controller paths
+- JSON-based configuration
+- Flexible controller registration options
+
+#### Updated Controller Paths
+All core controllers now use standardized bundle prefixes:
+
+**Admin Controllers**  
+
+| Old Path                         | New Path                                              |
+|----------------------------------|-------------------------------------------------------|
+| `slug`                           | `@sylius/admin-bundle/slug`                           |
+| `taxon-slug`                     | `@sylius/admin-bundle/taxon-slug`                     |
+| `taxon-tree`                     | `@sylius/admin-bundle/taxon-tree`                     |
+| `delete-taxon`                   | `@sylius/admin-bundle/delete-taxon`                   |
+| `product-attribute-autocomplete` | `@sylius/admin-bundle/product-attribute-autocomplete` |
+| `product-taxon-tree`             | `@sylius/admin-bundle/product-taxon-tree`             |
+| `save-positions`                 | `@sylius/admin-bundle/save-positions`                 |
+| `compound-form-errors`           | `@sylius/admin-bundle/compound-form-errors`           |
+| `tabs-errors`                    | `@sylius/admin-bundle/tabs-errors`                    |
+
+**Shop Controller**  
+`api-login` → `@sylius/shop-bundle/api-login`
+
+#### New Configuration System
+
+**Configuration Files**
+```text
+assets/
+  admin/
+    controllers.json  # Admin controller configurations
+  shop/
+    controllers.json  # Shop controller configurations
```

Comment:
```suggestion
    controllers.json  # Shop controller configurations
  controllers.json # Shared controllers configurations imported via flex
```

If any plugin/bundle come with controllers, they are added into `assets/controllers.json`

User are free to move theme into admin or shop if they want. However, after each `composer command` they will appear back inside `assets/controllers.json`